frontend/src/components/core/Callout.tsx

```diff
@@ -1,12 +1,20 @@
 import { SparklesIcon } from "@heroicons/react/24/outline";
 
 export default function Callout({
-    children
+    children,
+    color
 } : {
-    children?: React.ReactNode | React.ReactNode[]
+    children?: React.ReactNode | React.ReactNode[],
+    color: "green" | "blue" | "purple"
 }) {
+    const colorOptions = {
```

Comment:
We have three colors for the callout now! These all match the theme. At some point need to add the ability to change icon and toggle icon on / off so this can use used elsewhere.